src/test/java/edu/kis/powp/jobs2d/events/SelectTestFigureOptionListener.java

```diff
@@ -16,6 +24,27 @@ public SelectTestFigureOptionListener(DriverManager driverManager) {
 
 	@Override
 	public void actionPerformed(ActionEvent e) {
-		FiguresJoe.figureScript1(driverManager.getCurrentDriver());
+		DrawerFeature.getDrawerController().clearPanel();
+
+		switch (e.getActionCommand()){
+			case "Figure Joe 1":
+				DrawerFeature.getDrawerController().clearPanel();
+				FiguresJoe.figureScript1(driverManager.getCurrentDriver());
+				break;
+			case "Figure Joe 2":
+				DrawerFeature.getDrawerController().clearPanel();
+				FiguresJoe.figureScript2(driverManager.getCurrentDriver());
+				break;
+			case "Square":
+				DrawerFeature.getDrawerController().clearPanel();
+				FactoryManager factory1 = new DrawSquare();
+				factory1.create().execute(driverManager.getCurrentDriver());
+				break;
+			case "Rectangle":
```

Comment:
Menu action should not depend on its caption. Enum or separate listeners would be better. At least you should consider constants.